app/src/main/res/values/strings.xml

```diff
@@ -952,6 +952,34 @@
     <string name="hko_warning_text_tsunami" translatable="false">Tsunami Warning</string>
     <string name="hko_warning_text_fire_yellow_description" translatable="false">The Fire Danger Warning is Yellow and the fire risk is High.</string>
     <string name="hko_warning_text_fire_red_description" translatable="false">The Fire Danger Warning is Red and the fire risk is Extreme.</string>
+    <!-- Weather texts for MGM -->
+    <string name="mgm_weather_text_clear" translatable="false">@string/common_weather_text_clear_sky</string>
+    <string name="mgm_weather_text_slightly_cloudy" translatable="false">@string/common_weather_text_mainly_clear</string>
+    <string name="mgm_weather_text_partly_cloudy" translatable="false">@string/common_weather_text_partly_cloudy</string>
+    <string name="mgm_weather_text_mostly_cloudy" translatable="false">@string/common_weather_text_cloudy</string>
+    <string name="mgm_weather_text_rain_light" translatable="false">@string/common_weather_text_rain_light</string>
+    <string name="mgm_weather_text_rain" translatable="false">@string/common_weather_text_rain</string>
+    <string name="mgm_weather_text_rain_heavy" translatable="false">@string/common_weather_text_rain_heavy</string>
+    <string name="mgm_weather_text_sleet" translatable="false">@string/weather_kind_sleet</string>
+    <string name="mgm_weather_text_snow_light" translatable="false">@string/common_weather_text_snow_light</string>
+    <string name="mgm_weather_text_snow" translatable="false">@string/common_weather_text_snow</string>
+    <string name="mgm_weather_text_snow_heavy" translatable="false">@string/common_weather_text_snow_heavy</string>
+    <string name="mgm_weather_text_rain_showers_light" translatable="false">@string/common_weather_text_rain_showers_light</string>
+    <string name="mgm_weather_text_rain_showers" translatable="false">@string/common_weather_text_rain_showers</string>
+    <string name="mgm_weather_text_rain_showers_heavy" translatable="false">@string/common_weather_text_rain_showers_heavy</string>
+    <string name="mgm_weather_text_rain_showers_local">Local rain showers</string>
+    <string name="mgm_weather_text_hail" translatable="false">@string/weather_kind_hail</string>
+    <string name="mgm_weather_text_thunderstorm" translatable="false">@string/weather_kind_thunderstorm</string>
+    <string name="mgm_weather_text_thunderstorm_heavy">Heavy thunderstorm</string>
```

Comment:
Duplicate of ̀`openmeteo_weather_text_thunderstorm_with_heavy_hail`? Can be moved as a common text ;)

Reply:
Is it the same though? It may not be hailing. Let me check what other existing strings may fit, or check `openmeteo` definitions ;)

Reply:
Open-Meteo is based on the WMO Weather interpretation codes (WW) standard, so I believe all weather models use the same kind of language?

Reply:
In that case, it would be `metno_weather_text_andthunder`

Reply:
I'll just point it to `weather_kind_thunderstorm` to keep it simple.